__tests__/app.test.js

```diff
@@ -118,3 +118,88 @@ describe("GET /api/articles ", () => {
       });
   });
 });
+
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("responds with 200 and specific comment when given id", () => {
+      const commentToCompare =  [
+        {
+          comment_id: 16,
+          body: 'This is a bad article name',
+          article_id: 6,
+          author: 'butter_bridge',
+          votes: 1,
+          created_at: '2020-10-11T15:23:00.000Z'
+        }
+      ]
+  ;
+
+      return request(app)
+        .get("/api/articles/6/comments")
+        .expect(200)
+        .then(({ body }) => {
+            
+          const { comments } = body;
+        
+          expect(comments).toEqual(commentToCompare);
+        });
+    });
+
+    test("responds with 200 and specific comments when given id ordered by newest created first", () => {
+      const commentsToCompare =  [
+        {
+          comment_id: 14,
+          body: 'What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge.',
+          article_id: 5,
+          author: 'icellusedkars',
+          votes: 16,
+          created_at: '2020-06-09T05:00:00.000Z'
+        },
+        {
+          comment_id: 15,
+          body: "I am 100% sure that we're not completely sure.",
+          article_id: 5,
+          author: 'butter_bridge',
+          votes: 1,
+          created_at: '2020-11-24T00:08:00.000Z'
+        }
+      ];
+
+      return request(app)
+        .get("/api/articles/5/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toEqual(commentsToCompare);
+          expect(comments).toBeSortedBy("created_at");
+        });
+    });
```

Comment:
This test is really thorough as you're checking you get the exact comment objects you'd expect.

You could make it a little shorter as you have already checked you get back an array of comment objects, by just checking each comment has an article_id value of 5.

No need to change it, just something to consider for the future!